assets/ignition-configs/99-worker-ssh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 99-worker-ssh
+  labels:
+    machineconfiguration.openshift.io/role: master
```

Comment:
so one limitation of the MCS running in bootstrap mode is that it can only ever serve up master configs, not worker configs (https://github.com/openshift/machine-config-operator/blob/93e5cf868f70ebe9129ef945fb95626b16203c21/pkg/server/bootstrap_server.go#L59). Any machineconfig that we want to serve up needs to have the master label so it can be served up.